genai/example_test.go

```diff
@@ -26,13 +26,34 @@ import (
 	"net/http"
```

Comment:
Since this is now a generated file, it should have the standard DO NOT EDIT comment after the license.

Reply:
Never mind, I see it in another PR.